stylesheet.css

```diff
@@ -1,11 +1,75 @@
 html {
     box-sizing: border-box;
+    font-family: 'Source Sans Pro';
+    scroll-behavior: smooth;
 }
 
 * {
     box-sizing: inherit;
+    margin:0px;
+    padding: 0px;
 }
+html {
+    background-color:#484f4f;
+    color: ghostwhite;
```

Comment:
You can add these styles to the `html` style declaration at the top so you don't need to declare it twice.